spec/book_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rspec'
+require_relative '../Book/book'
+
+RSpec.describe Book do
+  describe '#can_be_archived?' do
+    it 'returns true if parent returns true' do
+      book = Book.new('No Starch Press', 'good', '2001-01-01')
+      test_method = book.can_be_archived?
+      expect(test_method).to eq(true)
+    end
+  end
+
+  describe '#can_be_archived?' do
+    it 'returns false if parent returns false' do
+      book = Book.new('No Starch Press', 'good', '2022-01-01')
+      test_method = book.can_be_archived?
+      expect(test_method).to eq(false)
+    end
+  end
```

Comment:
- [ ] @team Great work adding tests for the label class, however, it's still missing tests for initialize and to_json methods, kindly consider adding them.